ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>I believe the genre feature is just a string (not a list). If so, I believe the description should be "The genre of the movie" (not genres)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>In this NB, I use singular to simplify the data generation. But in the original dataset [1], it is plural because one movie can be categorized as more than one genre. Do you think we can let this slide? </p><p> </p><p>[1] https://files.grouplens.org/datasets/movielens/ml-25m-README.html</p>

Reply:
I see. If I understand correctly, we're treating the column value as a single string even in instances where there are multiple genres, ex: "Comedy|Romance"? Is this a best practice? If not, what should users do instead? Maybe we should add something like "Note: For simplicity, this notebook treats the genres column values as a string even though a value might include multiple genres. For example, a movie might have a `genres` feature value of "Comedy|Romance," which will be imported as a single string. In real-world scenarios, you might set the `genres` feature type to an array and transform the column to an array before importing the data."

Reply:
<p>In this specific notebook, one movie is always categorized as one genre: I fake the data so that each movie only has one genre and define the schema as such. So no "Comedy|Romance": I just pick one of them in such cases. Again just for simplicity purpose and the data is purely faked, according to the movie recommendation story.</p><p> </p><p>In the future, we can re-generate dataset and re-write the notebook accordingly if we decide to make the data more accurate/authentic.</p>